specification/network/resource-manager/Microsoft.Network/stable/2024-03-01/virtualNetwork.json

```diff
@@ -2089,6 +2094,18 @@
       },
       "description": "DhcpOptions contains an array of DNS servers available to VMs deployed in the virtual network. Standard DHCP option for a subnet overrides VNET DHCP options."
     },
+    "PrivateEndpointVNetPolicies": {
+      "type": "string",
+      "description": "Private Endpoint VNet Policies.",
+      "enum": [
+        "Disabled",
+        "Basic"
```

Comment:
Im assuming we will add the other enums in the future when they are supported, correct?

Reply:
Correct. In the future, we will add new enum members.